REQUIREMENTS.md

```diff
@@ -45,27 +45,27 @@ For each test, provide:
 
 ### Purpose
 This list will enable us to make those tests selectable within the Agworld interface by your customers.
-The test codes will populate the “test suites” column in [example_submission.csv](https://github.com/semiosBIO/modus-integration/blob/main/example_submission.csv)
+The test codes will populate the “test suites” column in [example_submission_barcodes.csv](https://github.com/semiosBIO/modus-integration/blob/main/example_submission_barcodes.csv)
 (i.e. we’ll put the id/code in there based on the user's selection. This can have multiple values which will be semi-colon separated.)
 
 There is a drop down list in Agworld that users can select from to pick the test suite(s) or individual test(s) that they want performed by you:
 
 ![In Agworld these options will be provided to your customers](https://github.com/semiosBIO/modus-integration/blob/main/lab_test_selection.png)
 
 How this dropdown relates to this step - we need a list of information to build this drop down list.
-Here's a (tiny) example:
+Here's a (tiny) :
 
 ![Send us this information](https://github.com/semiosBIO/modus-integration/blob/main/test_suite_csv.png)
 
-In the above example, the first three items are test suites, the last three are individual tests.
+In the above , the first three items are test suites, the last three are individual tests.
 
 The purposes of the four columns are as follows:
 
 **Code** is what you would like us to send to you in our CSV of sample data as the identifier of the test suite (or individual test) to perform, according to your business' own naming.
 
 **Display Name** is the name of the test suite (or individual test) that you would like visible to the user in the drop down list as seen above.
 
-**Description** is what is shown in the tooltip when the user hovers over the test name - which may describe (for example) the list of analyses performed as part of the test suite. For individual tests, it may only contain just the element name again.
+**Description** is what is shown in the tooltip when the user hovers over the test name - which may describe (for ) the list of analyses performed as part of the test suite. For individual tests, it may only contain just the element name again.
```

Comment:
```suggestion
**Description** is what is shown in the tooltip when the user hovers over the test name - which may describe (for example) the list of analyses performed as part of the test suite. For individual tests, it may only contain just the element name again.
```